### PR TITLE
Add support for enableRemoteStore parameter in opensearch-cluster-cdk

### PR DESCRIPTION
**Pull request by rishabh6788**
### Description
This PR adds support for enableRemoteStore parameter in opensearch-cluster-cdk.
This will be used to run nightly benchmarks on remote-store enabled clusters.

### Issues Resolved
https://github.com/opensearch-project/OpenSearch/issues/8087

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: vars/runBenchmarkTestScript.groovy [hunk 1]
```diff
@@ -62,6 +63,7 @@ void call(Map args = [:]) {
             args.singleNode.toBoolean() ? "--single-node" : "",
             args.minDistribution.toBoolean() ? "--min-distribution" : "",
             args.use50PercentHeap.toBoolean() ? "--use-50-percent-heap" : "",
+            args.enableRemoteStore.toBoolean() ? "--enable-remote-store": "",
```
Comment (peterzhuamazon) [inline vars/runBenchmarkTestScript.groovy:66]: Add space between `"--enable-remote-store"` and `:`.

**Review by peterzhuamazon — COMMENTED** [review]
Hi @rishabh6788, one nitpik here, thanks.

File: vars/runBenchmarkTestScript.groovy (hunk 1 above)
Comment (rishabh6788) [reply vars/runBenchmarkTestScript.groovy]: fixed.

**Review by peterzhuamazon — APPROVED** [review]

**Review by peterzhuamazon — CHANGES_REQUESTED** [review]
Block for now as it needs a version bump from another PR.

**Review by peterzhuamazon — APPROVED** [review]

**Pull request merged**